Makefile

```diff
@@ -167,6 +185,8 @@ release: tag_major_minor
 	docker push $(NAME)/node-base:$(VERSION)
 	docker push $(NAME)/node-chrome:$(VERSION)
 	docker push $(NAME)/node-firefox:$(VERSION)
+	docker push $(NAME)/node-opera:$(VERSION)
+	docker push $(NAME)/node-phantomjs:$(VERSION)
```

Comment:
Why is `node-phantomjs` here? This image was deprecated a while ago.

Reply:
Deleted.